app/javascript/elm/src/Main.elm

```diff
@@ -527,6 +546,33 @@ update msg model =
             in
             ( { model | overlay = Overlay.update overlay model.overlay }, Cmd.none )
 
+        UpdateResolution ->
+            ( model, Ports.updateResolution (51 - BoundedInteger.getValue model.crowdMapResolution) )
+
+        Timeout int ->
+            if int == model.debouncingCounter then
+                update UpdateResolution { model | debouncingCounter = 0 }
```

Comment:
Will it work if you just have here 
`( { model | debouncingCounter = 0 }, Ports.updateResolution (51 - BoundedInteger.getValue model.crowdMapResolution) )` ? 
And then you can remove `UpdateResolution` branch completely.